files/modrinth-exclude-include.json

```
@@ -83,6 +84,7 @@
     "sorted_enchantments",
     "visuality",
     "VR-Combat",
+    "welcome-screen",
```

and this one would be


```suggestion
    "welcomescreen",
```